tests/kafkatest/services/kafka/kafka.py

```diff
@@ -764,6 +769,9 @@ def prop_file(self, node):
             else:
                 override_configs[config_property.ZOOKEEPER_SSL_CLIENT_ENABLE] = 'false'
 
+        if self.use_new_coordinator:
+            override_configs[config_property.NEW_GROUP_COORDINATOR_ENABLE] = 'true'
```

Comment:
I don't see where we update the kafka config based on this flag. Is it missing?

Reply:
didn't get committed the first time, it's there yep